### PR TITLE
ashahba/Hotfix for 'ngtf-cpu_ubuntu-bin-build'

### PR DESCRIPTION
**Pull request by ashahba**
We need PR, otherwise the `buildkite/ngtf-cpu-ubuntu-binary-tf ` test always fail on `r0.19` branch

### CODE REVIEW

File: test/ci/buildkite/ngtf-cpu_ubuntu-bin-build.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
   - command: |
       source /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv/bin/activate 
+      python3 build_ngtf.py --use_tensorflow_from_location /localdisk/buildkite-agent/prebuilt_tensorflow_1_15_0
```
Comment (sayantan-nervana) [inline test/ci/buildkite/ngtf-cpu_ubuntu-bin-build.yaml]: We run "build_ngtf.py" twice in this yaml? One with use_tensorflow_from_location one with use_prebuilt_tensorflow?

Comment (ashahba) [reply test/ci/buildkite/ngtf-cpu_ubuntu-bin-build.yaml]: Let me check

Comment (ashahba) [reply test/ci/buildkite/ngtf-cpu_ubuntu-bin-build.yaml]: We shouldn't need this line

File: test/ci/buildkite/ngtf-cpu_ubuntu-bin-build.yaml [hunk 2]
```diff
@@ -24,6 +24,8 @@
   
   - command: |
       source /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv/bin/activate 
+      pip install -U /localdisk/buildkite-agent/prebuilt_tensorflow_1_15_0/artifacts/tensorflow/tensorflow-1.15.0-cp36-cp36m-linux_x86_64.whl
```
Comment (shresthamalik) [inline test/ci/buildkite/ngtf-cpu_ubuntu-bin-build.yaml]: Not sure why we need to reinstall tf.

This should have installed
`python3 build_ngtf.py --use_prebuilt_tensorflow  --artifacts /local/ .... `

**Comment by ashahba** [comment]
Ok, so this one is going to pick up the same changes we put into #456 to get the tests passing.
I'll make the changes when that PR is merged.

**Comment by adk9** [comment]
Closing because this issue is irrelevant now. Please reopen if you feel otherwise.

**Pull request closed without merging**